Shared/Samples/Set up location-driven geotriggers/SetUpLocationDrivenGeotriggersView.swift

```diff
@@ -0,0 +1,332 @@
+            }
+        }
+        
+        /// Create a geotrigger monitor and start observing its notifications.
+        /// - Parameters:
+        ///   - feed: The `GeotriggerFeed` that is monitored for changes.
+        ///   - featureTable: The `ServiceFeatureTable` that contains the features
+        ///   to use in the `FeatureFenceParameters`.
+        ///   - bufferDistance: The `Double` buffer distance in meters to apply to
+        ///   the features when checking if an `FenceGeotrigger` condition is met.
+        ///   - fenceGeotriggerName: The name for the `FenceGeotrigger`.
+        private func makeGeotrigger(feed: GeotriggerFeed, featureTable: ServiceFeatureTable, bufferDistance: Double, fenceGeotriggerName: String) {
+            let fenceParameters = FeatureFenceParameters(
+                featureTable: featureTable,
+                bufferDistance: bufferDistance
+            )
+            
+            // The Arcade expression in the fence geotrigger returns the value for
+            // the "name" field of the feature that triggered the monitor.
+            let fenceGeotrigger = FenceGeotrigger(
+                feed: feed,
+                ruleType: .enterOrExit,
+                fenceParameters: fenceParameters,
+                messageExpression: ArcadeExpression(expression: "$fenceFeature.name"),
+                name: fenceGeotriggerName
+            )
+            
+            // Create and start the geotrigger monitor.
+            let geotriggerMonitor = GeotriggerMonitor(geotrigger: fenceGeotrigger)
+            Task.detached { [unowned self] in
+                do {
+                    try await geotriggerMonitor.start()
+                    geotriggerMonitors.append(geotriggerMonitor)
+                } catch {
+                    self.error = error
+                }
+            }
+            
+            // Handle posted geotrigger notifications.
+            notificationTaskHandle = Task {
+                for await notification in geotriggerMonitor.notifications {
+                    handleGeotriggerNotification(notification as! FenceGeotriggerNotificationInfo)
+                }
+            }
+        }
+        
+        /// Handle a notification posted by a geotrigger monitor when a fence geotrigger
+        /// condition has been met.
+        /// - Parameter fenceNotificationInfo: The `FenceGeotriggerNotificationInfo`
+        /// which provides information about the geotrigger monitor and the fence
+        /// geotrigger that was triggered.
+        private func handleGeotriggerNotification(_ fenceNotificationInfo: FenceGeotriggerNotificationInfo) {
+            // The feature name from the Arcade expression.
+            let featureName = fenceNotificationInfo.message
+            let fenceFeature = fenceNotificationInfo.fenceGeoElement as! ArcGISFeature
```

Comment:
Though for this sample's data it is safe to force unwrap, I recommend `guard let` at the top in case the users might have different data.